rename_all_deprecated_modules.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+# Copyright 2021 Coop IT Easy SCRL fs
+#   Carmen Bianca Bakker <carmen@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+"""CIE-specific script to run migration on all test databases. This script may
+not work for you if your server is different.
+"""
+
+import datetime
+import logging
+import os
+import subprocess
+
+_logger = logging.getLogger(__name__)
+
+
+def all_databases():
+    result = subprocess.run(["ociedoo", "list-db"], check=True, stdout=subprocess.PIPE)
+    output = result.stdout.decode("utf-8")
+    return [line for line in output.splitlines() if line]
+
+
+def filter_databases(databases):
+    return [database for database in databases if database.endswith("-test")]
+
+
+def now():
+    return datetime.datetime.now().strftime("%Y-%m-%dT%H-%m")
+
+
+def main():
+    os.chdir(os.path.dirname(__file__))
+    logging.basicConfig(level=logging.INFO)
+    databases = filter_databases(all_databases())
+    for database in databases:
+        new_database = f"{database}-renamedep-{now()}"
```

Comment:
Do we really need a backup ? Or is the idea to have `xxx-komunigi-test` always on old modules ?

Reply:
The convenience of creating a new database is that this script becomes repeatable. If it failed, you don't have to delete + recreate `xxx-komunigi-test`. You can simply do your changes and run the script again.

Reply:
🆗